README.org

```diff
@@ -13,37 +13,37 @@
 
 * Content                                                                            :TOC:noexport:
 - [[#quick-feature-overview][Quick Feature Overview]]
-   - [[#fancy-gifs][Fancy Gifs!]]
```

Comment:
Any idea why the indent is changed? I thought setting the `org-list-indent-offset` in the header line should take care of that.

Reply:
No idea, I tried with both loading, and not loading settings, `toc-org-max-depth: 2` gets applied, but the offset does not... I'll edit it in fundamental mode and push.

Reply:
org-toc is always using the global value. I've opened a PR for it: https://github.com/snosov1/toc-org/pull/75